sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -836,7 +836,7 @@ public virtual async Task StopProcessingAsync(CancellationToken cancellationToke
             await foreach (var partitionEvent in consumer.ReadEventsFromPartitionAsync(partitionId, startingPosition, ProcessingReadEventOptions, cancellationToken).ConfigureAwait(false))
             {
                 using DiagnosticScope diagnosticScope = EventDataInstrumentation.ClientDiagnostics.CreateScope(DiagnosticProperty.EventProcessorProcessingActivityName);
-                diagnosticScope.AddAttribute("kind", "server");
+                diagnosticScope.AddAttribute("kind", "consumer");
```

Comment:
nit:  Could use make use of the constants from the `DiagnosticProperty` class here to avoid hardcoding the strings directly?